submissions/oluwabusolamiEluyela/html/index.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
+<html>
+  <head>
+<meta charset "UTF-8">
+  <title>justify.com</title>
+  </head>
+
+  <body>
+    <header>
+      <div> 
+        <center><h1><a href="#">JUSTIFY</a></h1></center>
+      </div>
+      <nav>
+        <ul>
+	    <center><h2><a href="#">Hot Deals</a>
+            <a href="#">Gift Cards</a>
+            <a href="#">Sell</a><c/enter>
+            <a href="#">Book Rentals</a></h2></center>
+
+             <li><a href="#">New Releases</a></li><li> <a href="#">General Book Shelf</a></li><li> <a href="#">Today's Deals</a></li><li> <a href="#">Registration</a></li><li> <a href="#">Teachers Corner</a></li><li> <a href="#">Students Corner</a></li><li> <a href="#">Contact</a></li><li> <a href="#">Order Offline</a></li>
```

Comment:
This is not cool. Please ensure your codes are properly indented. And I mean for this whole page too.